content/cpp/concepts/math-functions/terms/atanh/atanh.md

```diff
@@ -0,0 +1,103 @@
+---
+Title: 'atanh()'
+Description: 'Computes the arc hyperbolic tangent(inverse hyperbolic tangent) of a number in radians.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`atanh()`** function in C++ returns the arc hyperbolic tangent (inverse hyperbolic tangent) of a number in `radians`.The area hyperbolic tangent is the inverse operation of the hyperbolic tangent.
+
+## Syntax
+
+```pseudo
+atanh(data_type x)
+```
+
+`x`: The value of mandatory argument in range `[-1,1]`.
+
+## Parameter
+
+This function accepts one mandatory parameter `x` which specifies the inverse hyperbolic angle in radian which lies in the range `[-1, 1]`. The parameter can be of `double`, `float`, or `long double` datatype.
+
+## Error Handling
+
+If the argument is not on the interval `[-1, +1]`, a range error occurs.
+
+If the argument is `±1`, a pole error occurs.
+
+If the argument is `±0`, it is returned `unmodified`.
+If the argument is `±1`, `±∞` is returned and `FE_DIVBYZERO` is raised.
+If |arg|>1, `NaN` is returned and `FE_INVALID` is raised.
+If the argument is `NaN`, `NaN` is returned.
+
+## Prototype [C++11 Standard]
```

Comment:
I don't think we need this section. It's at odds with all the other math entries.